framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/util/SsoDeployer.java

```diff
@@ -137,16 +137,8 @@ private static void createRolesAndUsers(String authUrl, String realm) {
         ssoApi.createRole(ADMIN);
         ssoApi.createRole(KIE_SERVER);
         ssoApi.createRole(REST_ALL);
-        ssoApi.createUser(DeploymentConstants.getWorkbenchUser(),
-                DeploymentConstants.getWorkbenchPassword(),
+        ssoApi.createUser(DeploymentConstants.getAppUser(),
+                          DeploymentConstants.getAppPassword(),
                 Arrays.asList(ADMIN, KIE_SERVER, REST_ALL));
```

Comment:
To improve readability of code in future, can you please format these lines. Thanks

Reply:
This is already formatted by the IDE automatically.
I can edit it to have it a single line.